src/commands/infra/generate.ts

```diff
@@ -5,9 +5,9 @@ import * as os from "os";
 import path from "path";
 import simpleGit from "simple-git/promise";
 import { logger } from "../../logger";
+import * as infra_common from "./infra_common";
```

Comment:
It is not super idiomatic to use underscores in JavaScript variable names.

Given that, and the fact that the rest of the code is using camelCase, consider using camelCase here (eg. “infraCommon”)